src/app/core/services/label.service.ts

```diff
@@ -14,6 +14,10 @@ const COLOR_DARKNESS_THRESHOLD = 0.184;
 const COLOR_DARK_TEXT  = '000000'; // Dark color for text with light background
 const COLOR_LIGHT_TEXT  = 'FFFFFF'; // Light color for text with dark background
 
+const SEVERITY_TITLE = 'Severity';
+const TYPE_TITLE = 'Bug Type';
```

Comment:
*BUG_TYPE_TITLE

Reply:
Should I remove the title? It might not be clear to the reader if its just "SEVERITY"

Reply:
hmm, @damithc what do you think?

Reply:
As per our convention is (see the `COLOR_...` names), it should be something like `DISPLAY_NAME_BUG_TYPE`, `DISPLAY_NAME_SEVERITY`,  ...